scripts/report_event.py

```diff
@@ -23,11 +23,13 @@
 from SignaliQ.client import Client
 from SignaliQ.model.CloudProviderEvent import CloudProviderEvent
 from SignaliQ.model.ProviderEventsUpdateMessage import ProviderEventsUpdateMessage
+from SignaliQ.model.CloudVM import CloudVM
+from SignaliQ.model.NetworkInterface import NetworkInterface
 
 __log__ = logging.getLogger(__name__)
 
 env_id = 180005401 # CHANGE THIS TO LOCAL iQ ENVIRONMENT ID
-vm_uuid = "421be947-4e42-3deb-e0e9-55ef78c3e16a" # CHANGE THIS TO LOCAL VM UUID
+vm_hwid = "00:50:56:9B:7C:76" # CHANGE THIS TO A LOCAL VM MAC ADDRESS, MUST USE ':' NOT '-' FOR NOW
```

Comment:
The latest Signal-iQ library should handle the dashes case now.

Reply:
Updated to use dashes.